packages/node_modules/@webex/plugin-meetings/src/media/detectSilence/videoSilence.js

```diff
@@ -0,0 +1,48 @@
+/*
+  Code influenced from:
+    - https://stackoverflow.com/questions/46543341/how-can-i-extract-the-preceding-audio-from-microphone-as-a-buffer-when-silence
+    - https://github.com/twilio/twilio-video.js/blob/master/lib/util/detectsilentvideo.js
+    - https://github.com/twilio/twilio-video.js/blob/master/lib/util/detectsilentaudio.js
+    - https://github.com/twilio/twilio-video.js/blob/master/lib/webaudio/detectsilence.js
+*/
+
+import {
+  DETECT_VIDEO_SILENCE_SAMPLE_HEIGHT,
+  DETECT_VIDEO_SILENCE_SAMPLE_WIDTH
+} from '../../constants';
+
+/**
+ * Check whether the current video frame is silent by selecting a 50x50
```

Comment:
this comment is a bit misleading, we're not selecting a 50x50 sample, but downscaling the whole frame to a 50x50 image.